packages/db/src/config.ts

```diff
@@ -38,6 +38,7 @@ export const getSecretConnectionConfig = async (
     password: dbAccessMeta.password,
     database: dbAccessMeta.database,
     port: dbAccessMeta.port ?? 5432,
+    ssl: true,
```

Comment:
Presumably necessary due to the updated provider?

Reply:
This got dragged into the v5 commit, but that doesn't make sense - this shouldn't matter for our internal code.... 

Let me look into it.